p2p/netaddress_test.go

```diff
@@ -70,23 +90,21 @@ func TestNewNetAddressString(t *testing.T) {
 }
 
 func TestNewNetAddressStrings(t *testing.T) {
-	addrs, errs := NewNetAddressStrings([]string{"127.0.0.1:8080", "127.0.0.2:8080"})
+	addrs, errs := NewNetAddressStrings([]string{
```

Comment:
Could we also test some invalid addresses? The current test just confirms the always expected case.